src/Storages/Rocksdb/StorageEmbeddedRocksdb.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include <ext/shared_ptr_helper.h>
+#include <Storages/IStorage.h>
+#include <Storages/TableLockHolder.h>
+#include <Disks/IDisk.h>
+
+#include <shared_mutex>
+
+#include "rocksdb/db.h"
+#include "rocksdb/table.h"
+
+namespace DB
+{
+
+class Context;
+
+class StorageEmbeddedRocksdb final : public ext::shared_ptr_helper<StorageEmbeddedRocksdb>, public IStorage
+{
+    friend struct ext::shared_ptr_helper<StorageEmbeddedRocksdb>;
+    friend class EmbeddedRocksdbBlockOutputStream;
+public:
+    std::string getName() const override { return "EmbeddedRocksdb"; }
+
```

Comment:
```suggestion
    bool supportsParallelInsert() const override { return true; }
```

BTW I guess it can be safely added